shared/django_apps/rollouts/migrations/0002_auto_20240223_1522.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.7 on 2024-02-23 15:22
+
+from django.db import migrations
+
+
+def seed_initial_features(apps, schema_editor):
+    FeatureFlag = apps.get_model("rollouts", "FeatureFlag")
+    FeatureFlagVariant = apps.get_model("rollouts", "FeatureFlagVariant")
+
+    list_repos_generator = FeatureFlag.objects.create(
+        name="list_repos_generator", proportion=0.0
+    )
+    FeatureFlagVariant.objects.create(
+        name="list_repos_generator_enabled",
+        feature_flag=list_repos_generator,
+        proportion=1.0,
+        value=True,
+        override_repo_keys=["github/codecov", "bitbucket/codecov", "gitlab/codecov"],
+    )
+
+    use_label_index_in_report_processing = FeatureFlag.objects.create(
+        name="use_label_index_in_report_processing", proportion=0.0
+    )
+    FeatureFlagVariant.objects.create(
+        name="use_label_index_in_report_processing_enabled",
+        feature_flag=use_label_index_in_report_processing,
+        proportion=1.0,
+        value=True,
+        override_repo_keys=[
+            "github/giovanni-guidini/sentry",
+            "github/giovanni-guidini/components-demo",
+        ],
+    )
```

Comment:
to seed these features: https://github.com/codecov/worker/blob/f0c512d48558ef6d9fb8b7fea1dec1f9e1ea431c/rollouts/__init__.py#L16-L35